testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -22,7 +22,7 @@ public class TestCoordinator : NetworkBehaviour
 {
     public const int PerTestTimeoutSec = 5 * 60; // seconds
 
-    public const float MaxWaitTimeoutSec = 20;
+    public const float MaxWaitTimeoutSec = 30;
```

Comment:
I'd be down to have different timeout values depending on the case. Having everything depend on the same val for all timeouts might not be the best. this timeout is used in the tests themselves, 30 seconds for inter process communication timeout is pretty long

Reply:
reverting